[32mINFO    [0m integration.ha_tests.test_async_replication:test_async_replication.py:86 Destroying second model
[32mINFO    [0m pytest_operator.plugin:plugin.py:903 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.4.5    unsupported  01:54:55Z

App                  Version  Status  Scale  Charm                Channel      Rev  Address         Exposed  Message
postgresql-k8s       14.13    active      3  postgresql-k8s                      0  10.152.183.234  no       Primary
postgresql-test-app           active      1  postgresql-test-app  latest/edge  263  10.152.183.210  no       received database credentials of the first database

Unit                    Workload  Agent      Address       Ports  Message
postgresql-k8s/0        active    executing  10.1.170.215         
postgresql-k8s/1*       active    executing  10.1.170.206         Primary
postgresql-k8s/2        active    executing  10.1.170.205         
postgresql-test-app/0*  active    idle       10.1.170.213         received database credentials of the first database

Offer              Application     Charm           Rev  Connected  Endpoint           Interface         Role
replication        postgresql-k8s  postgresql-k8s  0    0/0        replication        postgresql_async  requirer
replication-offer  postgresql-k8s  postgresql-k8s  0    1/1        replication-offer  postgresql_async  provider

[32mINFO    [0m pytest_operator.plugin:plugin.py:909 Juju error logs:

unit-postgresql-k8s-0: 01:39:47 ERROR unit.postgresql-k8s/0.juju-log pgdata folder not found in /var/lib/postgresql/data/pgdata
unit-postgresql-k8s-0: 01:39:55 ERROR unit.postgresql-k8s/0.juju-log pgdata folder not found in /var/lib/postgresql/data/pgdata
unit-postgresql-k8s-0: 01:53:09 ERROR unit.postgresql-k8s/0.juju-log Cluster upgrade failed, ensure pre-upgrade checks are ran first.

[32mINFO    [0m pytest_operator.plugin:plugin.py:991 Forgetting model main...